framework/eslint-local-rules/rules.ts

```diff
@@ -56,4 +57,50 @@ export default {
             };
         },
     },
-} satisfies Record<string, Rule.RuleModule>;
\ No newline at end of file
+    'url-checker': {
+        meta: {
+            docs: {
+                description: 'Checks the URLs in the comment-docs for accessibility'
+            },
+            schema: [],
+        },        
+        create: function (context: Rule.RuleContext) : Rule.RuleListener {
+            function checkProgramComment(program: ESTree.Program): void {
+                if (!program.comments) {
+                    return;
+                }
+
+                for (const c of program.comments.filter(c => c.type == 'Block')) {
+                    const matches = c.value.match(/\bhttps?:\/\/[\-A-Za-z0-9+&@#\/%?=~_|!:,.;]*[\-A-Za-z0-9+&@#\/%=~_|]/);
+                    if (!matches || matches.length == 0) {
+                        continue;
+                    }
+
+                    for (const url of matches) {
+                        try {
+                            const res = request('GET', url, { headers: { 'user-agent': 'libcurl/1.0.6'} });
+                            if (res.statusCode != 200) {
```

Comment:
Should we also include here the case of a redirect?

Reply:
From my point of view, if there's a redirect we should change the URL to the target one. @vgkowski @dzeno your opinions?

Reply:
+1 with @balabad